docs/develop/frontend-dev.md

```diff
@@ -1,24 +1,49 @@
-# Running the Frontend for Development
+# Developing the Frontend UI
 
-This guide explains how to deploy an instance of the Browsertrix Cloud frontend for development.
+This guide explains how to run the Browsertrix Cloud frontend development server with [Yarn](https://classic.yarnpkg.com).
 
-This setup may be ideal for rapid development on the frontend, as it will allow running the frontend
-via `yarn` instead of the frontend image deployed in Kubernetes.
+Instead of rebuilding the entire frontend image to view your UI changes, you can use the included local development server to access the frontend from your browser. This setup is ideal for rapid UI development that does not rely on any backend changes.
 
 ## Requirements
 
-This setup does require a Browsertrix Cloud API backend already in a Kubernetes cluster.
+### 1. Browsertrix Cloud API backend already in a Kubernetes cluster
 
-Refer to [setup for local deployment](./local-dev-setup.md) for additional details on this setup.
+The frontend development server requires an existing backend that has been deployed locally or is in production. See [Deploying Browsertrix Cloud](https://docs.browsertrix.cloud/deploy/).
+
+### 2. Node.js >=16 and Yarn 1
+
+To check if you already have Node.js installed, run the following command in your command line terminal:
+
+```sh
+node --version
+```
+
+You should see a version number like `v18.12.1`. If you see a command line error instead of a version number, or your version is less than 16, [install or upgrade Node.js](https://nodejs.org) before continuing.
```

Comment:
```suggestion
You should see a version number like `v18.12.1`. If you see a command line error instead of a version number, or your version is less than 16, [install or upgrade Node.js](https://nodejs.org/) before continuing. If you need to use earlier versions of Node.js for different projects, [consider installing nvm](https://github.com/nvm-sh/nvm).
```

Reply:
IMO nvm seems a bit out of scope for this guide. Maybe this could be moved to an aside?

Reply:
![Uploading Screenshot 2023-08-15 at 9.50.00 AM.png…]()

Reply:
yea, i guess its ok to keep, but really we should assume users have basic familiarity with running node, this isn't a beginner tutorial. Also, node 16 is fairly old at this point :)